World Crime Rates Analysis/Model/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Follow the README template and fill up the README file as per your project. Try to represent it in such a way that it looks a bit catchy, as README files are gonna be the face of any project so you need to put extra effort to represent it nicely.

Reply:
Yeah I am on that, the data analysis notebook is updated, in the model folder, could you guide me as what else can i do in that.... so that I could do the README file all together...

Reply:
Other than that only the readme file updation is left..